app/routes.py

```diff
@@ -1 +1,235 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Flask, Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from datetime import date
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+#TASK ROUTES
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{model_id} is not a valid type ({type(model_id)}). Must be an integer)"}, 400))
+
+    model = cls.query.get(model_id)
+    
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} {model_id} does not exist"}, 404))
+        
+    return model
```

Comment:
Nice job creating this reusable function & keeping your code DRY. Nice approach to handling an invalid model 😁